application/config/features.php

```diff
@@ -62,8 +62,8 @@
 	],
 
 	// Post locking
-	'post-locking' => [
```

Comment:
@willdoran sorry, why is the post-locking feature removed and targeted survey replacing it?

Reply:
ugh, stupidity

Reply:
fixed